frontend/src/pages/TermPlanner/DraggableCourse.jsx

```diff
@@ -95,9 +94,9 @@ function DraggableCourse({ code, index }) {
 	  and the course has no warning, then disable the tooltip */}
       {!isDragDisabled && (warning1 || warning2) && (
         <ReactTooltip id={code} place="bottom" className="tooltip">
-          {isLegacy ? "This course is discontinued." : 
+          {isLegacy ? "This course is discontinued. If an equivalent course is currently being offered, please pick that instead." : 
             !isUnlocked ? prereqDisplay : 
-              warningMessage != "" ? warningMessage : handbook_note}
+              warningMessage !== "" ? warningMessage : handbook_note}
```

Comment:
whoaa huge respect for getting my super cooked logic here 😂  tbf, is there any other way of dealing with this not involving ternary operators?

Reply:
Ill be honest - this is also the code i would have written lmao